### PR TITLE
add Python 3 as build dep for Xvfb 1.20.8 easyconfigs

### LINKED ISSUE #10744: Xvfb has a build dependency on Python 3
I see this build error when Python 3 isn't available (yes, I know...):
```
== 2020-06-03 09:07:19,624 run.py:203 DEBUG run_cmd: running cmd  ./configure --prefix=/apps/eb/skylake/software/Xvfb/1.20.8-GCCcore-8.3.0  --build=x86_64-pc-linux-gnu  --host=x86_64-pc-linux-gnu --with-xkb-rules-symlink=xorg (in /dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29)
== 2020-06-03 09:07:19,624 run.py:222 INFO running cmd:  ./configure --prefix=/apps/eb/skylake/software/Xvfb/1.20.8-GCCcore-8.3.0  --build=x86_64-pc-linux-gnu  --host=x86_64-pc-linux-gnu --with-xkb-rules-symlink=xorg
== 2020-06-03 09:07:20,826 build_log.py:169 ERROR EasyBuild crashed with an error (at easybuild/base/exceptions.py:124 in __init__): cmd " ./configure --prefix=/apps/eb/skylake/software/Xvfb/1.20.8-GCCcore-8.3.0  --build=x86_64-pc-linux-gnu  --host=x86_64-pc-linux-gnu --with-xkb-rules-symlink=xorg" exited with exit code 1 and output:
checking for a BSD-compatible install... /bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking for a sed that does not truncate output... /bin/sed
checking for xsltproc... /bin/xsltproc
checking for a sed that does not truncate output... (cached) /bin/sed
checking for a Python interpreter with version >= 3.0... none
configure: error: no suitable Python interpreter found
 (at easybuild/tools/run.py:533 in parse_cmd_output)
```

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

fixes #10744

### CODE REVIEW

File: easybuild/easyconfigs/x/Xvfb/Xvfb-1.20.8-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -12,6 +12,7 @@ toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
 builddependencies = [
     ('binutils', '2.32'),
     ('pkg-config', '0.29.2'),
+    ('Python', '3.7.2'),
```
Comment (verdurin) [inline easybuild/easyconfigs/x/Xvfb/Xvfb-1.20.8-GCCcore-8.3.0.eb:15]: @boegel shouldn't this be `3.7.4`?

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/606d446aab7f6c9a7349101b521f2b97 for a full test report.

**Comment by verdurin** [comment]
@boegel I don't think we addressed my earlier report about OS deps for font handling, when this was merged.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3401.kirlia.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/ecd368c7179002bdf73769dafa1dc665 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2428.golett.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/2633141896b2015db9fcd667e9bf1cbc for a full test report.

**Comment by verdurin** [comment]
Also seeing a build failure on a different node:
```
== 2020-06-03 14:58:17,183 run.py:203 DEBUG run_cmd: running cmd  make -j 24  (in /dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29)
== 2020-06-03 14:58:17,183 run.py:222 INFO running cmd:  make -j 24
== 2020-06-03 14:58:17,294 build_log.py:169 ERROR EasyBuild crashed with an error (at easybuild/base/exceptions.py:124 in __init__): cmd " make -j 24 " exited with exit code 2 and output:
Making all in compat
make[1]: Entering directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/compat'
make[1]: Nothing to be done for `all'.
make[1]: Leaving directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/compat'
Making all in geometry
make[1]: Entering directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/geometry'
Making all in digital_vndr
make[2]: Entering directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/geometry/digital_vndr'
make[2]: Nothing to be done for `all'.
make[2]: Leaving directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/geometry/digital_vndr'
Making all in sgi_vndr
make[2]: Entering directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/geometry/sgi_vndr'
make[2]: Nothing to be done for `all'.
make[2]: Leaving directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/geometry/sgi_vndr'
make[2]: Entering directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/geometry'
make[2]: Nothing to be done for `all-am'.
make[2]: Leaving directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/geometry'
make[1]: Leaving directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/geometry'
Making all in keycodes
make[1]: Entering directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/keycodes'
Making all in digital_vndr
make[2]: Entering directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/keycodes/digital_vndr'
make[2]: Nothing to be done for `all'.
make[2]: Leaving directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/keycodes/digital_vndr'
Making all in sgi_vndr
make[2]: Entering directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/keycodes/sgi_vndr'
make[2]: Nothing to be done for `all'.
make[2]: Leaving directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/keycodes/sgi_vndr'
make[2]: Entering directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/keycodes'
make[2]: Nothing to be done for `all-am'.
make[2]: Leaving directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/keycodes'
make[1]: Leaving directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/keycodes'
Making all in po
make[1]: Entering directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/po'
make[1]: Nothing to be done for `all'.
make[1]: Leaving directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/po'
Making all in rules
make[1]: Entering directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/rules'
Making all in compat
make[2]: Entering directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/rules/compat'
./map-variants.py --want=mls --number=2 base.ml2_s.part ./layoutsMapping.lst ./variantsMapping.lst
make[2]: ./map-variants.py: Command not found
make[2]: *** [base.ml2_s.part] Error 127
make[2]: Leaving directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/rules/compat'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/dev/shm/Xvfb/1.20.8/GCCcore-8.3.0/xkeyboard-config-2.29/rules'
make: *** [all-recursive] Error 1
 (at easybuild/tools/run.py:533 in parse_cmd_output)
```

**Comment by verdurin** [comment]
@boegel any ideas on the build failure I'm seeing here?

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/dabd9f641b9cf38c5eeef221d8d0baed for a full test report.

**Comment by verdurin** [comment]
I think this at the top of `map-variants.py` is not helping:
```
#!/usr/bin/python3
```

**Comment by verdurin** [comment]
@boegel this has been addressed in `2.30` of `xkeyboard-config` so please add that, and fix the X11 fonts OS dep issue mentioned above.

**Comment by verdurin** [comment]
@boegel any updates on the issues raised above?

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/f0e666f7df1bfd4e4edf5ed1817dbc19 for a full test report.

**Comment by verdurin** [comment]
@boegel still seeing this error:
```
checking for a sed that does not truncate output... /usr/bin/sed
checking for root directory for font files... /eb/maint/software/Xvfb/1.20.8-GCCcore-8.3.0/share/fonts/X11
checking for directory for misc files... ${FONTROOTDIR}/misc
checking for fc-cache... /eb/maint/software/fontconfig/2.13.1-GCCcore-8.3.0/bin/fc-cache
checking for mkfontdir... no
configure: error: mkfontdir is required to build font-misc-misc.
 (at easybuild/tools/run.py:533 in parse_cmd_output)
```

Is there something else I need to rebuild?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3148.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/18985f6c1fce515e2645dc154401ba07 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/b997a23f840f61c491a8bb6a2712e310 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/47021b2c18e91eb5d3baf35b818d93ba for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/0b6343cd59abd480953198949dba8590 for a full test report.

**Comment by boegel** [comment]
@verdurin Looks like you need to reinstall `Mesa` (see #9129)

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3401.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/bfde8fb1e4135413bee099ac97cf4c58 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 24 out of 26 (2 easyconfigs in this PR)
nuc.lan - Linux Fedora 32, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.8.5
See https://gist.github.com/603a876625facfdb40193615f4114bf0 for a full test report.

**Comment by boegel** [comment]
@verdurin Please try again if the last commit fixes the problem you're seeing. I'm not sure, but found some clues online that suggest it could help.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/cb290370fe96c53c438ec04788e943d3 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3401.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/3a47b73334882e8464dd3969792e4663 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node2602.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/128d86d7cba3869635620cdd585cbcf1 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**